README.md

```diff
@@ -50,8 +50,8 @@ Use the Internet Identity canister in your local dfx project by adding the follo
   "canisters": {
     "internet_identity": {
       "type": "custom",
-      "candid": "https://github.com/dfinity/internet-identity/releases/download/release-2023-11-17/internet_identity.did",
-      "wasm": "https://github.com/dfinity/internet-identity/releases/download/release-2023-11-17/internet_identity_dev.wasm.gz",
+      "candid": "https://github.com/dfinity/internet-identity/releases/latest/download/internet_identity.did",
+      "wasm": "https://github.com/dfinity/internet-identity/releases/latest/download/internet_identity_dev.wasm.gz",
```

Comment:
This is not ideal as this doesn't pin the release. An alternative would be to update the README on new releases!

Reply:
why do we need to pin the release in the readme?

Reply:
people copy/paste it and then they get different versions depending on when they first ran their first `dfx` command; it's just best practice

Reply:
Added an automatic update: https://github.com/dfinity/internet-identity/pull/2226